code/iaas/model/src/main/java/io/cattle/platform/core/addon/BlkioDeviceOption.java

```diff
@@ -0,0 +1,61 @@
+package io.cattle.platform.core.addon;
+
+import io.github.ibuildthecloud.gdapi.annotation.Field;
+import io.github.ibuildthecloud.gdapi.annotation.Type;
+
+@Type(list = false)
+public class BlkioDeviceOption {
+
+    Integer readIops;
+    Integer writeIops;
+    Integer readBps;
+    Integer writeBps;
+    Integer weight;
+
+    @Field(required = false, nullable = true)
+    public Integer getReadIops() {
+        return readIops;
+    }
+
+    public void setReadIops(Integer readIOps) {
+        this.readIops = readIOps;
+    }
+
+    @Field(required = false, nullable = true)
```

Comment:
Nit: I believe required field is false by default; so this annotation can be skipped